docs/user/PostPostgreSQLDataTransfer_Experimental.md

```diff
@@ -30,4 +32,21 @@ You had an existing SAS Viya platform deployment that was created using the viya
       * If as part of your data transfer, you provisioned your PostgreSQL clusters without the use of a Viya 4 IAC projects, then you will need to manually add definitions for each of your PostgreSQL clusters. You can see an example definition here in our [CONFIG-VARS.md documentation](https://github.com/sassoftware/viya4-deployment/blob/main/docs/CONFIG-VARS.md#postgresql)
    2. If your PostgreSQL Cluster requires a certificate for connection, in your `ansible-vars.yaml` ensure that `V4_CFG_TLS_TRUSTED_CA_CERTS` is set and points to either the certificate or a directory containing that certificate.
    3. In your `ansible-vars.yaml` set `DEPLOY` to false
-   4. Run the ansible-playbook again with the `viya,install` tags. Since `DEPLOY` is set to false, the SAS Viya platform deployment will not be modified, however in your deployment directory you should see an updated kustomization.yaml with generated entries for your PostgreSQL clusters and those files should automatically be present in your site-config directory.
\ No newline at end of file
+   4. Run the ansible-playbook again with the `viya,install` tags. Since `DEPLOY` is set to false, the SAS Viya platform deployment will not be modified, however in your deployment directory you should see an updated kustomization.yaml with generated entries for your PostgreSQL clusters and those files should automatically be present in your site-config directory.
+
+#### Crunchy 4 
+
+1. While following the [PostgreSQL Data Transfer Guide](https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=default&docsetId=pgdatamig&docsetTarget=titlepage.htm) under "Steps for Crunchy Data 4" in Step 5 you are asked to remove all entries in your kustomization.yaml that contain a set of strings that are used for Crunchy 4 Data PostgreSQL configuration.
+2. Since the viya4-deployment project automatically manages the configuration and creation of the PostgreSQL related entries in your kustomziation.yaml, you can skip Step 5 from the "Steps for Crunchy Data 4" for now.
```

Comment:
I think #1 and #2 should combine as a step.  Currently, #1 states what is happening in the step 5 but they should not do that.  It would be more complete to combine them as a step.

Reply:
I consolidated those two steps, ready for re-review.